src/__tests__/mocks/handlers.ts

```diff
@@ -1,20 +1,16 @@
-import { addMocksToSchema } from "@graphql-tools/mock";
-import { makeExecutableSchema } from "@graphql-tools/schema";
 import { createHandler } from "../../handlers.js";
-import graphqlSchema from "../../../.storybook/stories/components/relay/schema.graphql";
-
-const schema = makeExecutableSchema({ typeDefs: graphqlSchema });
+import typeDefs from "../../../.storybook/stories/ecommerce-schema.graphql";
+import type { Resolvers } from "../../__generated__/resolvers-types-ecommerce.ts";
 
 const products = ["beanie", "bottle", "cap", "onesie", "shirt", "socks"];
 
-// Create a new schema with mocks
-const schemaWithMocks = addMocksToSchema({
-  schema,
+const graphQLHandler = createHandler<Resolvers>({
+  typeDefs,
   resolvers: {
     Query: {
       products: () =>
         Array.from({ length: products.length }, (_element, id) => ({
```

Comment:
I know this isn't part of this PR so feel free to ignore, but a potential optimization here might be to just `map` over `products` so you can avoid the whole array filling:

```ts
products.map((product, id) => ({
  id: `${id}`,
  title: product,
  // etc.
})
```